dashboard/app/models/pd/facilitator_program_registration.rb

```diff
@@ -87,7 +87,13 @@ def self.options
     ).freeze
   end
 
+  def clear_data
+    update!(form_data: {}.to_json)
+  end
+
   def validate_required_fields
+    return if user.try(:deleted?)
```

Comment:
it might be better to define a shared `system_deleted?` method on the form concern, make the required field validation dependent on that, and have the individual forms override the method with their various `user.try(:deleted?)` logics

Reply:
Done.